src/ocm/components/clusterConfiguration/review/ReviewOperatorsTable.tsx

```diff
@@ -15,43 +15,26 @@ export const ReviewOperatorsTable = ({ cluster }: { cluster: Cluster }) => {
   const { t } = useTranslation();
   const operatorNames = operatorLabels(t);
 
+  const operators = React.useMemo(
+    () => [OPERATOR_NAME_CNV, OPERATOR_NAME_ODF, OPERATOR_NAME_LVM],
```

Comment:
Ideally I'd want to have a type defined in a similar way to `SupportedPlatformType` that we can define in a single place and drives all operators. 

Therefore if we are missing an entry for an operator in such type, we'd get a TS error and we don't need to actively think about all places where we need to change when a new operator is added.

Reply:
I tried my best :slightly_smiling_face: